src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const widthSpider = spider.offsetWidth;
+  const borderWidth = Array.from(getComputedStyle(wall).borderTopWidth);
+  const borderWall = borderWidth
+    .filter((x) => x / x === 1 || x === '.')
+    .join('');
```

Comment:
This code is a bit unclear. It seems like you're trying to parse the border width as a number, but it's not very readable. Consider using the `parseInt` or `parseFloat` function to make your intention more clear.